ocrolib/common.py

```diff
@@ -279,7 +279,7 @@ def read_page_segmentation(fname):
     segmentation = make_seg_black(segmentation)
     return segmentation
 
-@checks(str,PAGESEG)
+@checks(str,_=PAGESEG)
```

Comment:
What is the meaning of this change? Help...

Reply:
checks is here https://github.com/tmbdev/ocropy/blob/ace19d5801c2dcd4d669ad4fac53b495e5077fe8/ocrolib/toplevel.py#L195

It expects positional arguments and keyword arguments that map argument names to types which are checked (I think). It can also check the **return** type agains the keyword argument `_` (https://github.com/tmbdev/ocropy/blob/ace19d5801c2dcd4d669ad4fac53b495e5077fe8/ocrolib/toplevel.py#L218).

So your change makes a check that `write_page_segmentation` returns `PAGESEG` (https://github.com/tmbdev/ocropy/blob/ace19d5801c2dcd4d669ad4fac53b495e5077fe8/ocrolib/toplevel.py#L428). Which it does not, it retuns `None`.

In short, I think you could just omit the `checks` decorator :whale:

Reply:
I don't follow here. AFAIK the change I make is deactivating the check for PAGESEG which is kind of conform to the other deactivated checks at this point. Thus, I think it should be okay to merge this PR. Or do you suggest to delete this line completely?

Reply:
I think the check does nothing so I would remove it.

Reply:
Okay, fine for me. There might be more of this sort of checks...

Reply:
Yeah, it's not ideal because the original check did make sense (checking that the second argument, `image` is one of `PAGESEG`.